.github/workflows/test-all-packages.yml

```diff
@@ -121,6 +115,24 @@ jobs:
         run: yarn run build:submissions
         working-directory: a3p-integration
 
+  # Verify that building the documentation works correctly.
+  lint-docs:
```

Comment:
We need to add a new `lint-docs` required check on all protected branches, right?

Reply:
> on all protected branches

More accurately: on all protected branches that have merged the current PR.

Reply:
Would it work as part of lint-rest? Just to save the repo admin activity.